app/assets/stylesheets/pin-show.css.scss

```diff
@@ -0,0 +1,117 @@
+$primary-background-color: rgba(237, 242, 242, 0.6);
+
+
+.container-fluid{
```

Comment:
I thought we can add and change thing own bootstrap classes?? this is the entire container bootstrap background color